### PR TITLE
use 44pt as the default minimum size for tappable elements

### LINKED ISSUE #6: Tappable area check not using Apple's recommendation
I noticed that the minimum tappable area check was failing for elements it should be passing for, according to [Apple's recommendation](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/), which is 44pt x 44 pt. 

It looks like the current default is 48pt x 48pt, which is a Material design standard. Since this is an iOS library, it makes most sense to follow Apple's rules by default.

Perhaps it would be a good idea to allow configuring rules like this one for the default checks.

### PR DESCRIPTION
**Pull request by niksawtschuk**
resolves #6.

### CODE REVIEW

**Comment by j-sid** [comment]
Thanks for all ur contributions Nik! We have used 48Points touch target size for historical reasons (as this project began to be used at Google), looking into approving ur changes.

**Comment by niksawtschuk** [comment]
@j-sid Any updates on getting this merged?

**Comment by j-sid** [comment]
**TL;DR Thank you for the change, and my apologies for the delay. Accepting this commit now.**

We had an internal discussion on 44pt as recommended by Apple design guidelines vs 48dp as recommended by Material (Google) design guidelines (and comparison of 44dp vs 44pt) as neither pt nor dp can be translated to physical units (inches/cm/mm etc) in across devices (each device will have its own conversion factor for pt/dp to physical units based on screen density). In addition we had to make sure that designers who use cross platform designs also have a easy way of specifying designs that can be cross platform. For now we have decided to go with 44X44pt when it comes to "minimum touch target that is accessible"

**Comment by niksawtschuk** [comment]
amazing, thanks @j-sid!
